Tests/Engine/GetScriptAnalyzerRule.tests.ps1

```diff
@@ -63,7 +63,7 @@ Describe "Test Name parameters" {
 
         It "get Rules with no parameters supplied" {
             $defaultRules = Get-ScriptAnalyzerRule
-            $expectedNumRules = 66
+            $expectedNumRules = 67
```

Comment:
note to self before merging: this will need to be bumped to 68 if #1806 already got merged.